src/controllers/onrampController.ts

```
@@ -94,6 +95,22 @@ export async function registerOnRampSwap(
+    logFinancialEvent({
+      event: "onramp.registered",
+      status: "pending",
+      transactionId: swap.id,
+      userId,
+      accountId: userWalletAddress,
+      idempotencyKey: swap.id,
+      amount: Math.round(xlmNum * 1e7), // XLM in stroops (7 decimal places)
+      currency: "XLM",
+      correlationId,
+      timestamp: new Date().toISOString(),
+      environment: (process.env.NODE_ENV ?? "development") as "production" | "staging" | "development",
```

ESLint

[error] Replace `·"production"·|·"staging"` with `⏎········|·"production"⏎········|·"staging"⏎·······`

(prettier/prettier)